lib/ts/recipe/session/utils.ts

```diff
@@ -55,17 +54,19 @@ export async function sendUnauthorisedResponse(
     __: BaseRequest,
     response: BaseResponse
 ) {
-    sendNon200Response(response, "unauthorised", recipeInstance.config.sessionExpiredStatusCode);
+    sendNon200ResponseWithMessage(response, "unauthorised", recipeInstance.config.sessionExpiredStatusCode);
 }
 
 export async function sendInvalidClaimResponse(
     recipeInstance: SessionRecipe,
-    validationErrors: ClaimValidationError[],
+    claimValidationErrors: ClaimValidationError[],
     __: BaseRequest,
     response: BaseResponse
 ) {
-    setInvalidClaimHeader(response, JSON.stringify(validationErrors));
-    sendNon200Response(response, "invalid claim", recipeInstance.config.invalidClaimStatusCode);
+    sendNon200Response(response, recipeInstance.config.invalidClaimStatusCode, {
+        message: "invalid claim",
+        claimValidationErrors,
+    });
```

Comment:
We should discuss the implications of this in the context of:
- Frameworks that don't have a response object upfront
- AWS lambda with custom authoriser
- Is there a possibility of the library or the user tryingt o send a JSON response again after this?
- What happens if session cookies are to be updated here. Will they get reflected in the response?

Reply:
All is fine, except that we need to confirm that sending a body is OK in other frameworks.